lib/internal/bootstrap_loaders.js

```diff
@@ -109,9 +109,12 @@
 
   const config = process.binding('config');
 
+  // Think of this as module.exports in this file even though it is not
+  // written in CommonJS style.
+  const exports_ = { internalBinding, NativeModule };
```

Comment:
Not generally a huge fan of trailing `_` variables in JS, esp since it looks similar enough to `exports`. Could this be like `loaderExports` or something?